frontend/device/android/device.lua

```diff
@@ -59,6 +59,7 @@ local external = require("device/thirdparty"):new{
         { "GoldenFree", "GoldenDict Free", false, "mobi.goldendict.android.free", "send" },
         { "GoldenPro", "GoldenDict Pro", false, "mobi.goldendict.android", "send" },
         { "Kiwix", "Kiwix", false, "org.kiwix.kiwixmobile", "text" },
+        { "Kiwix (FDroid)", "Kiwix (FDroid)", false, "org.kiwix.kiwixmobile.standalone", "text" },
```

Comment:
```suggestion
        { "KiwixStandalone", "Kiwix (F-Droid)", false, "org.kiwix.kiwixmobile.standalone", "text" },
```

Reply:
A bit of nitpicking :)

Reply:
I would prefeer F-Droid because these strings are not translated currently.

Also the menu will check those available and let you select among them, so it doesn't matter if it isn't obtained from f-droid.

Reply:
Okay.